integration-tests/integration_test.go

```diff
@@ -400,9 +400,11 @@ func (s *RepeatedNetworkFlowTestSuite) SetupSuite() {
 	err = s.collector.Launch()
 	s.Require().NoError(err)
 
+	scheduled_curls_image := "quay.io/rhacs-eng/qa:" + GetQATag("collector-schedule-curls")
```

Comment:
another slight nit but this pattern is repeated so might benefit from its own helper

```go
func QaImage(name string, tag string) string {
    return name + ":" + GetQaTag(tag)
}

QaImage("quay.io/rhacs-eng/qa", "collector-schedule-curls")
```

Reply:
Makes sense, I'll make this change.